tasks/serf.yml

```diff
@@ -18,11 +18,11 @@
     dest="{{serf_destination}}"
     mode=0750
     owner="{{smartstack_user}}"
-    group="{{smartstack_user}}" 
+    group="{{smartstack_user}}"
```

Comment:
should be `smartstack_group` and in the defaults add `smartstack_group = "{{ smartstack_user }}"` rather?